### PR TITLE
feat(meta): add CONTRIBUTING guide

### LINKED ISSUE #12: Add CONTRIBUTING file
Need good contribution guidelines, might as well be very explicit about expectations!

### PR DESCRIPTION
**Pull request by smartcontracts**
Adds a simple CONTRIBUTING guide so developers know what the heck is
going on.

Closes #12

### CODE REVIEW

**Comment by sam-goldman** [comment]
👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+### Testing
+
+```sh
+yarn test
+```
+
+## Conventional commits
+
+Please us the [conventional commit](https://www.conventionalcommits.org) format for commit messages.
+
```
Comment (sam-goldman) [inline CONTRIBUTING.md:77]: use*

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+yarn lint
+```
+
+### Testing
+
+```sh
+yarn test
+```
+
+## Conventional commits
```
Comment (sam-goldman) [inline CONTRIBUTING.md:74]: Perhaps it'd be helpful for new contributors to know the scopes that are commonly used in this project? For example, someone new might not know what "ct" means, and it isn't in the linked Conventional Commits page.

Comment (smartcontracts) [reply CONTRIBUTING.md:74]: Added

**Review by sam-goldman — APPROVED** [review]

**Pull request merged**